cmd/earthly/buildkit.go

```diff
@@ -79,9 +79,21 @@ func (app *earthlyApp) configureSatellite(cliCtx *cli.Context, cloudClient cloud
 
 	app.console.Warnf("") // newline
 	app.console.Warnf("The following feature flags are recommended for use with Satellites and will be auto-enabled:")
-	app.console.Warnf("--new-platform, --use-registry-for-with-docker")
+	app.console.Warnf("  --new-platform, --use-registry-for-with-docker")
 	app.console.Warnf("") // newline
 
+	if app.ci {
+		app.console.Warnf("When using Satellites, the --ci flag is an alias for:")
+		app.console.Warnf("  --strict --no-output when running earthly in target mode.")
+		app.console.Warnf("  --strict when running earthly in artifact or image mode.")
+		app.console.Warnf("") // newline
+		if !app.imageMode && !app.artifactMode {
+			app.noOutput = true
+		}
+		app.strict = true
+		app.ci = false
```

Comment:
Most of this kind of logic takes place in `actionBuildImp` under `build_cmd.go`. I would move this logic over there, to keep things in a single spot and prevent confusion.

(To detect whether you're on a satellite in there you could use this existing variable, which will cover both remote buildkit and satellites `isLocal := containerutil.IsLocal(app.buildkitdSettings.BuildkitAddress)`)